app/models/manageiq/providers/storage_manager/cinder_storage_manager/event_catcher.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::StorageManager::CinderStorageManager::EventCatcher < ::MiqEventCatcher
+  require_nested :Runner
+
+  def self.ems_class
+    ManageIQ::Providers::StorageManager::CinderStorageManager
+  end
+
+  def self.settings_name
+    :event_catcher_cinder_storage
+  end
+
+  # def self.all_valid_ems_in_zone
```

Comment:
I think we need this. In the setting.yml, we have to set the corresponding event catcher.